angularjs/angular-animate.d.ts

```diff
@@ -135,12 +142,120 @@ declare module angular.animate {
         classNameFilter(expression?: RegExp): RegExp;
     }
 
-	///////////////////////////////////////////////////////////////////////////
+    ///////////////////////////////////////////////////////////////////////////
     // Angular Animation Options
-	// see https://docs.angularjs.org/api/ngAnimate/#applying-directive-specific-styles-to-an-animation
-	///////////////////////////////////////////////////////////////////////////
-	interface IAnimationOptions {
-		to?: Object;
-		from?: Object;
-	}
+    // see https://docs.angularjs.org/api/ngAnimate/#applying-directive-specific-styles-to-an-animation
+    ///////////////////////////////////////////////////////////////////////////
+    interface IAnimationOptions {
+        /**
+         * to - The ending CSS styles (a key/value object) that will be applied across the animation via a CSS transition.
```

Comment:
you don't need the property names like `to - `

sorry, it's a little nitpicky, but let's be consistent with the other interfaces.